boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -79,15 +79,58 @@ if [ "$diff_generate" = true ] ; then
 else
     rm -rf "$SAAS_OPERATOR_DIR"
     git clone --branch "$operator_channel" ${GIT_PATH} "$SAAS_OPERATOR_DIR"
+
+
+    # PATH to saas file in app-interface
+    SAAS_FILE_URL="https://gitlab.cee.redhat.com/service/app-interface/raw/master/data/services/osd-operators/cicd/saas/saas-${operator_name}.yaml"
+
+    # MANAGED_RESOURCE_TYPE
+    # SAAS files contain the type of resources managed within the OC templates that
+    # are being applied to hive.
+    # For customer cluster resources this should always be of type "SelectorSyncSet" resources otherwise
+    # can't be sync'd to the customer cluster. We're explicity selecting the first element in the array.
+    # We can safely assume anything that is not of type "SelectorSyncSet" is being applied to hive only
+    # since it matches ClusterDeployment resources.
+    # From this we'll assume that the namespace reference in resourceTemplates to be:
+    # For customer clusters: /services/osd-operators/namespace/<hive shard>/namespaces/cluster-scope.yaml
+    # For hive clusters: /services/osd-operators/namespace/<hive shard>/namespaces/<namespace name>.yaml
+    MANAGED_RESOURCE_TYPE=$(curl -s "${SAAS_FILE_URL}" | \
+            docker run --rm -i quay.io/app-sre/yq:3.4.1 yq r - "managedResourceTypes[0]"
```

Comment:
`docker` will need to be conditioned. Something like [this](https://github.com/openshift/boilerplate/pull/142/files#diff-424ddf34b68cef0633f4520f756a919d70a24880b5ab931d3cd85d17f8659b5fR119-R126) :P

Reply:
Fixed.